### PR TITLE
Setting up RTD

### PR DESCRIPTION
**Pull request by trbromley**
https://pennylane-lightning.readthedocs.io/en/mocking_out_cpp/

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -165,6 +164,8 @@ def build_extensions(self):
     ),
 ]
 
+if os.environ.get("USING_RTD", False):
+    ext_modules = []
```
Comment (trbromley) [inline setup.py:168]: Am thinking to use this: https://docs.readthedocs.io/en/stable/guides/environment-variables.html

Comment (josh146) [reply setup.py:168]: @trbromley how come? Could it be useful to be able to compile the documentation locally even without the C++ requirements?

Comment (josh146) [reply setup.py:168]: Oh I suppose you could always set this locally! Perhaps you could make the env variable more general? `LQ_MOCK_DOCS`?

Comment (trbromley) [reply setup.py:168]: Nice idea!

**Pull request merged**